pkg/sessionctx/variable/session.go

```diff
@@ -3155,6 +3153,21 @@ type BatchSize struct {
 	MaxPagingSize int
 }
 
+// PipelinedDMLConfig defines the configuration for pipelined DML.
+type PipelinedDMLConfig struct {
+	// BulkDMLEnabled indicates whether to enable bulk DML in pipelined mode.
+	BulkDMLEnabled bool
+
+	// PipelinedFLushConcurrency indicates the number of concurrent worker for pipelined flush.
+	PipelinedFlushConcurrency int
+
+	// PipelinedResolveLockConcurrency indicates the number of concurrent worker for pipelined resolve lock.
+	PipelinedResolveLockConcurrency int
+
+	// PipelinedWriteThrottleRatio throttles the rate of pipelined flush. 1 = no throttle.
```

Comment:
Better to explain what the `throttle` indicates in details or give some examples in the comments.